tests/nm_utils/utils_skip.py

```diff
@@ -0,0 +1,134 @@
+"""Checks environment variables to skip various test groups.
+The functions here are imported by each test file.
+The .github/actions/nm-test-skipping-env-setup sets these 
+    variables in the testing automation.
+"""
+
+import os
+
+
+def should_skip_accuracy_test_group():
+    TEST_ACCURACY = os.getenv("TEST_ACCURACY", "ENABLE")
+    return TEST_ACCURACY == "DISABLE"
+
+
+def should_skip_async_engine_test_group():
+    TEST_ASYNC_ENGINE = os.getenv("TEST_ASYNC_ENGINE", "ENABLE")
+    return TEST_ASYNC_ENGINE == "DISABLE"
+
+
+def should_skip_basic_correctness_test_group():
+    TEST_BASIC_CORRECTNESS = os.getenv("TEST_BASIC_CORRECTNESS", "ENABLE")
+    return TEST_BASIC_CORRECTNESS == "DISABLE"
+
+
+def should_skip_core_test_group():
+    TEST_CORE = os.getenv("TEST_CORE", "ENABLE")
+    return TEST_CORE == "DISABLE"
+
+
+def should_skip_distributed_test_group():
+    TEST_DISTRIBUTED = os.getenv("TEST_DISTRIBUTED", "ENABLE")
+    return TEST_DISTRIBUTED == "DISABLE"
+
+
+def should_skip_engine_test_group():
+    TEST_ENGINE = os.getenv("TEST_ENGINE", "ENABLE")
+    return TEST_ENGINE == "DISABLE"
+
+
+def should_skip_entrypoints_test_group():
+    TEST_ENTRYPOINTS = os.getenv("TEST_ENTRYPOINTS", "ENABLE")
+    return TEST_ENTRYPOINTS == "DISABLE"
+
+
+def should_skip_kernels_test_groups():
+    TEST_KERNELS = os.getenv("TEST_KERNELS", "ENABLE")
+    return TEST_KERNELS == "DISABLE"
+
+
+def should_skip_lora_test_group():
+    TEST_LORA = os.getenv("TEST_LORA", "ENABLE")
+    return TEST_LORA == "DISABLE"
+
+
+def should_skip_metrics_test_group():
+    TEST_METRICS = os.getenv("TEST_METRICS", "ENABLE")
+    return TEST_METRICS == "DISABLE"
+
+
+def should_skip_model_executor_test_group():
+    TEST_MODEL_EXECUTOR = os.getenv("TEST_MODEL_EXECUTOR", "ENABLE")
+    return TEST_MODEL_EXECUTOR == "DISABLE"
+
+
+def should_skip_models_test_group():
+    TEST_MODELS = os.getenv("TEST_MODELS", "ENABLE")
+    return TEST_MODELS == "DISABLE"
+
+
+def should_skip_models_core_test_group():
+    TEST_MODELS_CORE = os.getenv("TEST_MODELS_CORE", "ENABLE")
+    return TEST_MODELS_CORE == "DISABLE"
+
+
+def should_skip_prefix_caching_test_group():
+    TEST_PREFIX_CACHING = os.getenv("TEST_PREFIX_CACHING", "0")
```

Comment:
The "0" here should be "ENABLE"?

Reply:
yes

Reply:
I’ve fixed this prior to my merge commit to resolve conflicts.